buildspec-ecr.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR...
+      - aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/uktrade
+  build:
+    commands:
+      - echo Build started on `date`
+      - echo Building the Docker image...
+      - docker build -t tunnel:latest --file Dockerfile.conduit .
+      - docker tag tunnel:latest public.ecr.aws/uktrade/tunnel:latest
+  post_build:
+    commands:
+      - echo Build completed on `date`
+      - echo Pushing the Docker image...
+      - docker push public.ecr.aws/uktrade/tunnel:latest
```

Comment:
Can we add the line break at the end to stop GitHub moaning?

You can probably automate this on save in your IDE.